src/sql/workbench/services/query/common/queryModel.ts

```diff
@@ -82,6 +82,7 @@ export interface IQueryModelService {
 	onRunQueryUpdate: Event<string>;
 	onRunQueryComplete: Event<string>;
 	onQueryEvent: Event<IQueryEvent>;
+	onConnIdAvailable: Event<QueryConnID>;
```

Comment:
nit: `onConnectionIdUpdated` would be more accurate IMO (this and other similar-named things in the other files)